repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/trainingProgrammePage-main.ftl

```diff
@@ -95,8 +95,16 @@
             <#--  Right hand content blocks: Table of content and content blocks   -->
             <aside class="page__rightbar">
                 <#--  Table of content  -->
-                <div class="hee-anchorlinks" data-toc-js="true">
-                    <h2 data-anchorlinksignore="true">Table of Contents</h2>
+                <div class="hee-hee-card hee-card--related-links theme__item-border">
```

Comment:
@FranciscoRuizHEE - Do you think we can remove this section (`Pages related to this programme`) as I guess it will be dealt separately as part of `NWPS-1556` ticket ?

Reply:
Removed the section as discussed with @sankar-shunmuga (as it would be dealt with separately as part of `NWPS-1556`)